common/src/main/java/com/wynntils/wynn/objects/profiles/ingredient/IngredientProfile.java

```diff
@@ -89,22 +88,22 @@ public ItemStack asItemStack() {
         ItemStack itemStack = ingredientInfo.asItemStack();
 
         if (itemStack.getItem() == Items.PLAYER_HEAD) {
-            HashMap<String, String> ingredientHeadTextures = Managers.ItemProfiles.getIngredientHeadTextures();
-
-            if (ingredientHeadTextures.containsKey(name)) {
-                CompoundTag skullData = new CompoundTag();
-                skullData.putString("Id", UUID.randomUUID().toString());
-
-                CompoundTag properties = new CompoundTag();
-                ListTag textures = new ListTag();
-                CompoundTag textureEntry = new CompoundTag();
-                textureEntry.putString("Value", ingredientHeadTextures.get(name));
-                textures.add(textureEntry);
-                properties.put("textures", textures);
-                skullData.put("Properties", properties);
-
-                itemStack.getOrCreateTag().put("SkullOwner", skullData);
-            }
+            String ingredientHeadTexture = Managers.ItemProfiles.getIngredientHeadTexture(name);
+            // This will look bad, but if we don't have the data, then what should we do?
```

Comment:
Not every ingredient texture is a head, so this comment is a bit weird.

Reply:
This is stated in the context of `if (itemStack.getItem() == Items.PLAYER_HEAD) {`. Maybe this looks weird in the diff, check out how it looks in the updated file. 

If the item is represented by item type PLAYER_HEAD and we have no texture, it sure is going to look bad. Ideally, I'd like a "hook" to notify us that we are missing essential data.